topics/transcriptomics/tutorials/minerva-pathways/tutorial.md

```diff
@@ -0,0 +1,327 @@
+</div>
+
+Now we're ready to analyse the counts files. Here we'll take the feature counts dataset collection and merge it into one count matrix through the use of "Column join". This can then be annotated with the human readable names of the genes. This is all passed to limma for differential expression analysis.
+
+With this result in hand we're ready to do two further steps: preparing the dataset for goseq, and for analysis in MINERVA. Goseq is a tool for gene ontology enrichment analysis, and MINERVA is a tool for visualising pathway analysis.
+
+<figure>
+{% assign wf_counts = site | get_workflow:page, "Galaxy-Workflow-mRNA-Seq_BY-COVID_Pipeline__Analysis.ga" %}
+<pre class="mermaid">
+{{ wf_counts.mermaid }}
+</pre>
+<figcaption><b>Workflow 2</b>: Analysis Workflow for mRNA-Seq BY-COVID Pipeline.</figcaption>
+</figure>
+
+> <hands-on-title>Analyse the Counts</hands-on-title>
+>
+> 1. Run the workflow with the Factor Data from the first Hands on, and the datasets from the workflow or Zenodo download, depending on your path:
+>
+>    {% snippet faqs/galaxy/workflows_run_wfh.md title="mRNA-Seq BY-COVID Pipeline" wfhub_id="689" box_type="none" %}
+>
+{: .hands_on}
+
+You should have a few outputs, namely the `goseq` outputs, and a table ready for visualisation in MINERVA!
+
+## MINERVA
+
+The dataset prepared for MINERVA must be correctly formatted as a tabular dataset (`\t` separated values) like the following, with the dbkey set to `hg19` or `hg38`. If you've run the above workflow, this should be the case.
+
+```
+SYMBOL  logFC              P.Value               adj.P.Val
+TRIM25  2.07376444684004   1.2610025125617e-18   3.57368112059986e-15
+ACSL1   2.90647033200259   2.71976234791064e-16  3.85390324698937e-13
+NBEAL2  2.45952426389725   2.71787290816654e-14  2.56748394058132e-11
+MIR150  -2.55304226607428  9.55912390273625e-14  6.74866152827879e-11
+SLC2A3  2.95861349227708   1.19066011437523e-13  6.74866152827879e-11
+```
+
+> <hands-on-title>Visualise in MINERVA</hands-on-title>
+>
+> 1. **Click** to expand the final "MINERVA-Ready Table"
+>
+> 1. Click on the {% icon galaxy-barchart %} (**Visualize**) icon
+>
+> 1. Select "display at Minerva (SARS-CoV-2 Minerva Map)"
+>
+>    > <tip-title>MINERVA not listed?</tip-title>
+>    > The MINERVA visualisation is only for correctly formatted files with the correct genome (i.e. human, hg19).
+>    > If you dont' see MINERVA listed, first check that your dataset is:
+>    > 1. recognised as a tabular dataset
+>    >
+>    >    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="tabular" %}
+>    >
+>    > 2. Has the correct genome build:
+>    >
+>    >    {% snippet faqs/galaxy/datasets_change_dbkey.md dbkey="hg19" %}
+>    >
+>    >    It should be specifically `hg19` not a patch like `hg19Patch5`
+>    >
+>    > If that still doesn't work, please check that the Galaxy server you are using is updated to 24.0 or later.
+>    >
+>    {: .tip}
+>
+{: .hands_on}
+
+### Analysis
+
+
+TODO
```

Comment:
@mjostaszewski @MattiHoch  if y'all can write something on how to do the data interpretation with MINERVA that would be great (which buttons to click, what do you suggest to look at (e.g. a pathway or component or two that the user should open and examine some data in the left hand menu or so)